script.js

```diff
@@ -0,0 +1,82 @@
+window.addEventListener('load', init);
+document.addEventListener('keydown', print);
+document.addEventListener('keyup', () => {
+    document.querySelectorAll('.keyboard__key').forEach(element => {
+        if (element.innerText != 'CapsLock') element.classList.remove('keyboard__key--active');
+    })
+})
+
+const keyValue = [
+    '`', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '-', '=', 'Backspace',
+    'Tab', 'q', 'w', 'e', 'r', 't', 'y', 'u', 'i', 'o', 'p', '[', ']', '\\', 'Del',
+    'CapsLock', 'a', 's', 'd', 'f', 'g', 'h', 'j', 'k', 'l', ';', '\'', 'Enter',
+    'Shift', '\\', 'z', 'x', 'c', 'v', 'b', 'n', 'm', ',', '.', '/', 'Up', '.Shift',
+    'Ctrl', 'Win', 'Alt', 'Space', 'Alt', 'Ctrl', 'Left', 'Down', 'Right'
+];
+
+function init() {
+    // Создание области вывода
+    let out = document.createElement('textarea');
+    out.className = 'output';
+    out.setAttribute('id', 'output');
+    document.body.append(out);
+
+    // Создание "обертки-контейнера" для клавиатуры
+    let board = document.createElement('div');
+    board.className = 'keyboard';
+    document.body.append(board);
+
+    // Создание кнопок
+    for (let i = 0; i < keyValue.length; i++) {
+        let key = document.createElement('button');
+        key.innerText = keyValue[i];
+        key.className = 'keyboard__key';
+        board.append(key);
+        if (['Backspace', 'Enter', 'Shift', 'CapsLock'].indexOf(key.innerText) !== -1) key.classList.add('keyboard__key--wide');
```

Comment:
here in each for cycle iteration you create new array with button names, it's preferably to create variable with such array and use it here, so less memory will be used